Formula/coreos-ct.rb

```diff
@@ -0,0 +1,56 @@
+class CoreosCt < Formula
+  desc "Convert a Container Linux Config into Ignition"
+  homepage "https://coreos.com/os/docs/latest/configuration.html"
+  url "https://github.com/coreos/container-linux-config-transpiler/archive/v0.6.1.tar.gz"
+  sha256 "57cafa9db08caa6bed0290612bebf5d1a763a96fdb21f23de1681fd266b0c3c3"
+  head "https://github.com/coreos/container-linux-config-transpiler"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    bin.install "./bin/ct"
+  end
+
+  test do
+    assert_equal "ct ", shell_output("#{bin}/ct -version").chomp
+
+    input_file = Tempfile.new("ct-input")
+    input_file.write(test_input)
+    input_file.close
+    assert_equal test_output.strip, shell_output("#{bin}/ct -pretty -in-file #{input_file.path}").strip
```

Comment:
```
/ssh_authorized_keys:.*- ssh-rsa mykey/m
```

Reply:
Sorry, I'm just a bit confused on what you mean by this?

Reply:
meaning delete the test_output function and use

```
    output = shell_output("#{bin}/ct -pretty -in-file #{input}")
    assert_match /ssh_authorized_keys:.*- ssh-rsa mykey/m, output
```

Reply:
Test seems to be failing on my local machine (also switched `#{input}` to `#{testpath}/input` -- no variable named `input`), will have to look into it a little deeper...

Reply:
The test failed on my local machine, but I was able to update it so that it works!